test/integration/base_integration_test.h

```diff
@@ -169,6 +189,12 @@ class BaseIntegrationTest : protected Logger::Loggable<Logger::Id::testing> {
       const Protobuf::int32 expected_error_code = Grpc::Status::WellKnownGrpcStatus::Ok,
       const std::string& expected_error_message = "");
 
+  // Wait for a batch of requests to arrive. This is useful if the order of
+  // expected requests is non-deterministic.
+  AssertionResult compareMultipleDeltaDiscoveryRequests(
```

Comment:
I can't spot where this is used exactly..

Reply:
It was used in a previous iteration of this PR (for the LEDS integration test where more than one locality LEDS stream is used, and the order of responses was non-deterministic).
Removed as it is not needed for the current tests.